src/Layout/Header.story.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Breadcrumbs, DropdownButton, DropdownLink, Link, StatusIndicator } from "..";
+import Header from "./Header";
+import Summary from "../Summary/Summary";
+import SummaryItem from "../Summary/SummaryItem";
+import SummaryDivider from "../Summary/SummaryDivider";
+import DropdownMenu from "../DropdownMenu/DropdownMenu";
+
+export const DefaultBackground = () => (
+  <Header
+    breadcrumbs={
+      <Breadcrumbs>
+        <Link href="/">Home</Link>
+        <Link href="/">Materials</Link>
+      </Breadcrumbs>
+    }
+    title="Materials Overview"
+  >
+    <StatusIndicator type="informative">Beta</StatusIndicator>
+  </Header>
+);
+
+export const UnstyledBackground = () => (
+  <Header
+    unstyledBackground
+    breadcrumbs={
+      <Breadcrumbs>
+        <Link href="/">Home</Link>
+        <Link href="/">Materials</Link>
+      </Breadcrumbs>
+    }
+    title="Materials Overview"
+  >
+    <StatusIndicator type="informative">Beta</StatusIndicator>
+  </Header>
+);
+
+export const CustomBackground = () => (
+  <Header
+    breadcrumbs={
+      <Breadcrumbs>
+        <Link href="/">Home</Link>
+        <Link href="/">Materials</Link>
+      </Breadcrumbs>
+    }
+    title="Materials Overview"
+    background="#98FB98"
+    unstyledBackground
+  >
+    <StatusIndicator type="informative">Beta</StatusIndicator>
+  </Header>
+);
+
+export const WithSubTitle = () => (
+  <Header
+    breadcrumbs={
+      <Breadcrumbs>
+        <Link href="/">Home</Link>
+        <Link href="/">Materials</Link>
+      </Breadcrumbs>
+    }
+    title="Materials Overview"
+    subtitle="Materials Overview"
+  >
+    <StatusIndicator type="informative">Beta</StatusIndicator>
+  </Header>
+);
+
+export const WithSummary = () => (
+  <Header
+    breakpoints={{
+      small: 1200,
+      medium: 1200,
+    }}
+    breadcrumbs={
+      <Breadcrumbs>
+        <Link href="/">Home</Link>
+        <Link href="/">Materials</Link>
+      </Breadcrumbs>
+    }
+    title="Materials Overview"
+    subtitle="Materials Overview"
+    summary={
+      <Summary>
```

Comment:
Also, using `React.cloneElement` we will get hidden props like `collapseBreakpoint`. Comparing to render prop we can define a type of a function.